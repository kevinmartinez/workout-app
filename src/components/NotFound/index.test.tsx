import React from 'react'
import { render, RenderResult } from '@testing-library/react'
import { NotFound } from '.'

let documentBody: RenderResult

describe('<NotFound />', () => {
  beforeEach(() => {
    documentBody = render(<NotFound />)
  })

  it('shows not found message', () => {
    expect(documentBody.getByText(/Not Found/i)).toBeInTheDocument()
    expect(documentBody.getByText('404')).toBeInTheDocument()
  })
})

describe('<NotFound />', () => {
  beforeEach(() => {
    documentBody = render(<NotFound />)
  })
  it('matches snapshot', () => {
    const { baseElement } = documentBody
    expect(baseElement).toMatchSnapshot()
  })
})
