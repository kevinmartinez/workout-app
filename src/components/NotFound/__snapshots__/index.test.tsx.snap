// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound /> matches snapshot 1`] = `
<body>
  <div>
    <div>
      404
    </div>
    <div>
      File Not Found
    </div>
  </div>
</body>
`;
